index/scorch/merge.go

```diff
@@ -129,8 +146,65 @@ func (s *Scorch) parseMergePlannerOptions() (*mergeplan.MergePlanOptions,
 	return &mergePlannerOptions, nil
 }
 
+type mergerKick struct {
+	moptions *mergeplan.MergePlanOptions
+	done     chan struct{}
```

Comment:
not sure if this is common bleve internals naming convention, but perhaps the ch suffix... like "doneCh"?

Reply:
oops..I missed this. thanks @steveyen !